pkgs/development/python-modules/django-webpush/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, pytest-django
+, python
+, pythonOlder
+, pywebpush
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "django-webpush";
+  version = "0.3.4";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
We have a new parameter for the format.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters